src/components/CatalogSearchCriteria.tsx

```diff
@@ -77,6 +77,8 @@ export const CatalogSearchCriteria = (props: Props) => (
           <input
             value={props.dateFrom}
             type="text"
+            pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}"
```

Comment:
Nice!  Hadn't considered using the `pattern` property.  It _used_ to be `<input type="date">` but as QA pointed out a few months back, Firefox hates that input type. :trollface: